includes/gateways/class-wpsms-gateway-adpdigital.php

```diff
@@ -63,9 +63,9 @@ public function SendSMS()
         $to  = str_replace("09", "989", implode(",",$this->to));
         $msg = urlencode($this->msg);
 
-        $result = file_get_contents("{$this->wsdl_link}send?username={$this->username}&password={$this->password}&dstaddress={$to}&body={$msg}&clientid={$this->from}&type=text&unicode=1");
+        $result = $this->request('GET', "{$this->wsdl_link}send?username={$this->username}&password={$this->password}&dstaddress={$to}&body={$msg}&clientid={$this->from}&type=text&unicode=1");
```

Comment:
FYI: method `$this→request` the throwable and should be in `try/catch` @foadadeli 

Solution: set `$throwFailedHttpCodeResponse=false`